portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro

```diff
@@ -0,0 +1,195 @@
+definition {
+
+	macro _addSubfolderInAnExistingStructuredContentFolder {
+		Variables.assertDefined(parameterList = "${name}");
+
+		var portalURL = JSONCompany.getPortalURL();
+
+		if (!(isSet(userEmailAddress))) {
+			var userEmailAddress = "test@liferay.com";
+		}
+
+		if (!(isSet(userPassword))) {
+			var userPassword = "test";
+		}
+
+		if (!(isSet(description))) {
+			var description = "";
+		}
+
+		if (!(isSet(externalReferenceCode))) {
+			var externalReferenceCode = "";
+		}
+
+		if (!(isSet(name))) {
+			var name = "";
+		}
+
+		if (!(isSet(viewableBy))) {
```

Comment:
is this field required for the body? do we fill it in at any point?

Reply:
This is not reuqired, I've force pushed to remove the viewableBy filed.